### DISCUSSION TITLE
How to config that COMPLETED MO didn't adopt yield?

### DISCUSSION THREAD

**Discussion by sgfisme** [frePPLe/frepple#441] [Q&A] [answered]
I set up sub-operations and adopt yield on the last operation. I put the WIP on the last operation and set its status as COMPLETED using manufacturing order, but the WIP still adopt the yield. If I hope it didn't adopt yield, how can I config my WIP using manufacturing order?

**Comment by jdetaeye** [comment, marked as answer]
If the operation is completed you should know the real quantity produced by the operation.  Until the actual yield of the production order is known, I think it's correct to work with the average expected yield.  So, "working as expected".

3 solutions to report the actual yield:
- Simple solution: set the WIP status to "closed", and include the correct produced quantity in the onhand.
- Cheating solution: set the WIP status to "completed", and adjust the production order quantity such that after applying the yield you end up with the correct produced quantity
- Complex solution:  set the WIP status to "completed" and also use an "operationplanmaterial/inventorydetail" record to specify  the actual produced quantity.
